sw/device/lib/testing/meson.build

```diff
@@ -0,0 +1,78 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+src_root_dir = meson.source_root()
+vendor_dir = src_root_dir / 'sw/vendor'
+gtest_src_dir = vendor_dir / 'google_googletest'
+gtest_build_dir = meson.current_build_dir() / 'google_googletest'
```

Comment:
Can you add a comment that spcifies exactly what directory this is relative to $REPO_TOP?

Reply:
Sure! It is `$REPO_TOP/build-out/sw/fpga/sw/device/lib/testing/google_googletest`, also added a comment.